stream-chat-swift-docs/docs/guides/getting-started.md

```diff
@@ -1,3 +1,149 @@
 ---
 title: Getting Started
----
\ No newline at end of file
+---
+
+## User Authentication
+
+After the SDK is installed, the next step is to initialize the chat client and authenticate the user. It can be a regular user, or it can be an [anonymous](https://getstream.io/chat/docs/anon/?language=swift) or [guest](https://getstream.io/chat/docs/guest_users/?language=swift) user with limited sets of permissions. A `TokenProvider` is the object that will hold your authentication strategy and will tell the `ChatClient` which user you're currently logged in with.
+
+## User Tokens
+
+User Tokens are JWT tokens containing a User ID and used to authenticate a user. You can [generate these tokens using a server client](https://getstream.io/chat/docs/ios-swift/tokens_and_authentication/?language=swift#generating-tokens) and then use them to authenticate a user in a chat client.
+
+### Regular User
+
+Regular users are the way to go for a most chat apps. They must be authenticated with a JWT token generated with your app's secret key. Ideally, you'll [generate the token in your backend](https://getstream.io/chat/docs/tokens_and_authentication/?language=swift) and provide a closure to fetch it, but for testing purposes we provide a [JWT generator](https://getstream.io/chat/docs/token_generator/?language=swift) and you can set the token statically. It's also possible to [use development tokens](https://getstream.io/chat/docs/node/tokens_and_authentication/#development-tokens), but they must be enabled for your app in the dashboard.
+
+#### Static
+
+```swift
+/// A static token provider. Use it for setting the token synchronously or for testing purposes.
+let token = Token("USER_TOKEN")
+let tokenProvider = TokenProvider.static(token)
+```
+
+#### Closure
+
+```swift
+/// A token provider that fetches the token from your backend service. Use it in production.
+let tokenProvider = TokenProvider.closure { chatClient, completion in
+    let token: Token?
+    let error: Error?
+
+    /// TODO: Fetch a token locally or use URLSession/Alamofire/etc to fetch
+    /// a token from your backend service and pass it into completion
+
+    if let token = token {
+        completion(.success(token))
+    } else if let error = error {
+        completion(.failure(error))
+    }
+}
+```
+
+#### Development
+
```

Comment:
"To use this, it must be enabled" as you've linked above, the info should be here